lib/glific/messages.ex

```diff
@@ -818,6 +828,19 @@ defmodule Glific.Messages do
     |> Repo.aggregate(:count)
   end
 
+  def add_send_by(messages) do
+    messages
+    |> Enum.map(fn message -> message |> append_send_by() end)
```

Comment:
Do we really need to traverse all the messages?

Can't we just add send_by in message_types and resolve there?

Reply:
With this approach I think it will work with conversation API only

Reply:
@pankaj-ag This is resolved. can you review it again?